node-zerox/src/utils.ts

```diff
@@ -104,3 +108,25 @@ export const convertPdfToImages = async ({
     console.error("Error during PDF conversion:", err);
   }
 };
+
+// Convert each page (from docx) to a png and save that image to tmp
+export const convertDocxToPdf = async ({
+  localPath,
+  tempDir,
+}: {
+  localPath: string;
+  tempDir: string;
+}): Promise<string> => {
+  const inputBuffer = await fs.readFile(localPath);
+  const outputFilename = path.basename(localPath, ".docx") + ".pdf";
```

Comment:
We can probably pass in the extension from where we call this function, so that it works with multiple file type (not just .docx).

```
export const convertToPdf = async ({
  extension,
  localPath,
  tempDir,
}: {
  extension: string;
  localPath: string;
  tempDir: string;
}): Promise<string> => {
  const inputBuffer = await fs.readFile(localPath);
  const outputFilename = path.basename(localPath, extension) + ".pdf";
  const outputPath = path.join(tempDir, outputFilename);

```